scripts/download-test-apps.sh

```diff
@@ -0,0 +1,18 @@
+# INSTALL SAUCECTL
+curl -L https://saucelabs.github.io/saucectl/install | bash
+
+# GET URL'S FOR SAMPLE REAL DEVICE APP AND XCTEST APP FROM GITHUB
+app_repo=saucelabs/my-demo-app-ios
+echo "Using sample apps from https://github.com/saucelabs/my-demo-app-ios/releases" 
+latest_tag=$(curl -s "https://api.github.com/repos/$app_repo/releases/latest" | grep -o '"tag_name": ".*"' | sed 's/"tag_name": "//' | sed 's/"$//')
```

Comment:
Optional: Using `jq` to parse JSON might be better?

```suggestion
latest_tag=$(curl -s "https://api.github.com/repos/$app_repo/releases/latest" | jq -r '.tag_name')
```

Reply:
That'd be something the user would have to install in addition to. `jq` isn't something that's available by default, I think.

Reply:
It's not available by default. I've seen people have problems with JQ not being available and demo snippets not working.